Cargo.toml

```diff
@@ -87,7 +87,7 @@ regex = "1.3"
 serde = { version = "1.0", features=["derive"] }
 serde_json = "1.0"
 serde_urlencoded = "0.6.1"
-time = "0.1.42"
+time = "0.2.1"
```

Comment:
If you're not using any deprecated methods, you can use `time = { version = "0.2.2", default-features = false, features = ["std"] }`.